CONTRIBUTING.md

```diff
@@ -1,11 +1,9 @@
 ## Adding a new tutorial
 
-See the `example` directory for an example.
-
 1. Create a new top-level directory (e.g. `my_tutorial_dir`)
 2. Add a file called `.notebooks` to your tutorial directory and add the base name of each tutorial script/notebook pair (e.g. `my_tutorial`) as a separate line in `.notebooks`
 3. Add a `requirements.txt` to your directory if additional ones are needed
-4. Add a command to `[testenv]` in `tox.ini` by copying `example` and add the `requirements.txt` file if necessary. Also add the command name to `envlist`.
+4. Add a command to `[testenv]` in `tox.ini` by copying `spam` and add the `requirements.txt` file if necessary. Also add the command name to `envlist`.
```

Comment:
to be fair, spam is probably one of the more complex examples

Reply:
it's just the lines in `tox.ini`